src/commonMain/kotlin/com/algolia/search/model/response/ResponseLogs.kt

```diff
@@ -84,7 +91,11 @@ public data class ResponseLogs(
         @SerialName(KeyIndex) val indexNameOrNull: IndexName? = null,
         @SerialName(Key_Exhaustive_Nb_Hits) val exhaustiveNbHits: Boolean? = null,
         @SerialName(Key_Exhaustive_Faceting) val exhaustiveFaceting: Boolean? = null,
-        @SerialName(KeyQuery_Params) val queryParamsOrNull: String? = null
+        @SerialName(KeyQuery_Params) val queryParamsOrNull: String? = null,
+        /**
+         * Contains an object for each performed query with the indexName, queryID, offset, and userToken.
+         */
+        @SerialName(KeyInner_Queries) val innerQueries: List<InnerQuery>? = null
```

Comment:
What do you think of the `OrNull` idiom? I see here we used it for `String` and `IndexName` but not `Boolean`
Then you would do below:
```
public val innerQueries: List<InnerQuery>
            get() = innerQueriesOrNull!!
```

Reply:
`orNull` it should be used to express variance. Usually `xOrNull` exists when there is a similar `x` that throws when something is missing or otherwise goes wrong. 
AFAIK, there is no logic that may throw in this particular case, I am right ?

Reply:
Thanks for the clarification. 
The list of inner queries is optional in this case, declaring it as nullable, is the correct approach then.